test/marketplace/MarketPlace.sol

```diff
@@ -46,24 +47,26 @@ contract MarketPlace {
   
   /// @param s Address of the deployed swivel contract
   function setSwivelAddress(address s) external onlyAddress(admin) returns (bool) {
+    require (!swivelInitiated, 'can only set the swivel address once`);
```

Comment:
we can require that the adress 'swivel' == `0x0` in order to be set (meaning: it hasn't been set yet). once it has this check will fail. use the .sol default instead of an extra bool

Reply:
Yea thats the way better solution. Good call.

Reply:
done. #179